packages/schemas/tables/connectors.sql

```diff
@@ -0,0 +1,12 @@
+create type connector_type as enum ('SMS', 'Email', 'Social');
+
+create table connectors (
+  id varchar(128) not null,
+  identifier varchar(256) not null unique,
+  enable boolean not null default TRUE,
+  type connector_type not null,
+  config jsonb /* @use ConnectorConfig */ not null default '{}'::jsonb,
+  data jsonb /* @use ConnectorData */ not null default '{}'::jsonb,
```

Comment:
looks like these two fields will be updated at the same time. maybe `config` is enough?

Reply:
`config`: updated at admin console by management API (developer).
`data`: updated at connector runtime by connector itself, for instance: update template review status.

But merge them into a single `config` is also acceptable, any idea? @simeng-li

Reply:
things like template review status should be a real-time fetching stuff?

Reply:
make sense, remove `data` for now.

Reply:
merge +1